test/potluck_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_is_an_empty_array
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
```

Comment:
nice job testing the attributes here (date and dishes)

Reply:
totally fine to break these tests apart like you did, or to do it like i showed you above where you roll them into one `test_it_has_attributes` and include multiple assertions. whatever makes more sense to you.